packages/scripts/src/utils/votingv2-utils.ts

```diff
@@ -62,14 +62,33 @@ export const getUniqueVoters = async (votingV2: VotingV2Ethers): Promise<string[
 
 export const updateTrackers = async (votingV2: VotingV2Ethers, voters: string[]): Promise<void> => {
   console.log("Updating trackers for all voters");
-  const batchSize = 25;
+  const priceRequests = await votingV2.callStatic.getNumberOfPriceRequestsPostUpdate();
+  const batchSize = 10;
   const batches = [];
   for (let i = 0; i < voters.length; i += batchSize) {
     const batch = voters.slice(i, i + batchSize);
     batches.push(
-      votingV2
-        .multicall(batch.map((voter) => votingV2.interface.encodeFunctionData("updateTrackers", [voter])))
-        .then((tx) => tx.wait())
+      (async () => {
```

Comment:
This LGTM! So the theory is that this is causing errors because it's trying to use too much gas in the read call, which is causing an invalid json rpc response? I'm just a little surprised -- I would've expected a more specific error about gas (as would be typical on a non-mainnet fork).